pkg/clientutil/client.go

```diff
@@ -83,24 +74,16 @@ func newClientWithPlatform(cmd *cobra.Command, platform string, clientOpts ...co
 		platformM := platforms.Only(platformParsed)
 		clientOpts = append(clientOpts, containerd.WithDefaultPlatform(platformM))
 	}
-	return newClient(cmd, clientOpts...)
+	return NewClient(ctx, namespace, address, clientOpts...)
 }
 
-// getDataStore returns a string like "/var/lib/nerdctl/1935db59".
+// GetDataStore returns a string like "/var/lib/nerdctl/1935db59".
 // "1935db9" is from `$(echo -n "/run/containerd/containerd.sock" | sha256sum | cut -c1-8)`
 // on Windows it will return "%PROGRAMFILES%/nerdctl/1935db59"
-func getDataStore(cmd *cobra.Command) (string, error) {
-	dataRoot, err := cmd.Flags().GetString("data-root")
-	if err != nil {
-		return "", err
-	}
+func GetDataStore(dataRoot, address string) (string, error) {
```

Comment:
```suggestion
func DataStore(dataRoot, address string) (string, error) {
```

https://go.dev/doc/effective_go#Getters

Reply:
Done